src/components/ChatMessage/ChatMessage.js

```diff
@@ -41,7 +43,19 @@ export default class ChatMessage extends React.Component {
       isSettingColumnVisibility: false,
       activeMenu: undefined,
       localRTFilterResponse: null,
+      isQueryOutputModalVisible: false,
+      // Add resize state
```

Comment:
Some of these comments from line 47 to 57 are not very useful or the variable name is descriptive enough. I would consider removing some of them.